src/components/poll.tsx

```diff
@@ -9,9 +9,9 @@ import {
 import { FunctionComponent, useContext, useEffect, useState } from "react";
```

Comment:
I actually didn't change this, seems like it was a result of Prettier formatting. There is a lot of code that could be refactored from this module. Based on my previous comments I would start by abstracting the API to it's own module.